### PR TITLE
[MM-21979] Fix incorrect placeholder prop for searchable user list

### PR DESCRIPTION
**Pull request by fm2munsh**
#### Summary
Placeholder is expected to be of type 
```
    placeholder: {
        id: string;
        defaultMessage: string;
        values?: {string: any};
    };
 ```
however there was a recent commit changing the type being passed down to a string causing any component referencing `SearchableUserList` to error on load

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-21979

### CODE REVIEW

**Comment by fm2munsh** [comment]
Requesting @jgilliam17 for review since the original PR (https://github.com/mattermost/mattermost-webapp/pull/4253) was also reviewed by them

**Comment by fm2munsh** [comment]
@jgilliam17 not sure why the first build failed but I created pushed a new commit and created a new test server for you

**Comment by jgilliam17** [comment]
> @jgilliam17 not sure why the first build failed but I created pushed a new commit and created a new test server for you

Thank you @fm2munsh

File: components/searchable_user_list/searchable_user_list.jsx [hunk 1]
```diff
@@ -226,11 +226,11 @@ class SearchableUserList extends React.Component {
                         id='searchUsersInput'
                         ref='filter'
                         className='form-control filter-textbox'
-                        placeholder={searchUsersTxt}
+                        placeholder={searchUsersPlaceholder}
```
Comment (sudheerDev) [inline components/searchable_user_list/searchable_user_list.jsx]: Shouldn't the placeholder also be translated?

**Pull request merged**